services/worker/src/worker/config.py

```diff
@@ -47,6 +50,10 @@ def from_env(cls) -> "WorkerConfig":
                 sleep_seconds=env.int(name="SLEEP_SECONDS", default=WORKER_SLEEP_SECONDS),
                 only_job_types=env.list(name="ONLY_JOB_TYPES", default=get_empty_str_list()),
                 storage_paths=env.list(name="STORAGE_PATHS", default=get_empty_str_list()),
+                state_path=env.str(name="STATE_PATH", default=None),
```

Comment:
I think we could move the default value WORKER_STATE_PATH here  "worker_state.json", almost all configs had their own default value

Reply:
`WORKER_STATE_PATH ` has no default - it changes for every sessions using a temporary directory.

Reply:
Yes, that's already what we do with the assets directory: https://github.com/huggingface/datasets-server/blob/main/libs/libcommon/src/libcommon/config.py#L14

Note: I prefer to always indirect via a constant, ie. `WORKER_STATE_PATH: Optional[str] = None`. Not sure if it's an exaggeration, though.

Reply:
Another comment: I didn't get that STATE_PATH is the state file basename (we create a file by appending `.lock` at the end at https://github.com/huggingface/datasets-server/pull/824/files#diff-643260fba42f231dbf4e91102b52af6acb3c216a0eb6f538ac42bc667ebe5381R145). Maybe the name could be more descriptive, or maybe we could replace by `STATE_FILENAME` and just use it without appending .lock?